doc/api/synopsis.md

```diff
@@ -88,6 +88,6 @@ Now, open any preferred web browser and visit `http://127.0.0.1:3000`.
 If the browser displays the string `Hello, World!`, that indicates
 the server is working.
 
-[Command-line options]: cli.md#command-line-options
+[Command-line options]: cli.md#options
```

Comment:
You renamed this to “Command-line API” elsewhere in this PR.

Reply:
Yes but I believe it makes more sense to target the `Options` section directly here.